### PR TITLE
Update LibertyUtils.findFileInWorkspace to use Path

### LINKED ISSUE #80: Fix bug with schema generation
Found during testing the early release for the Eclipse tooling that there is more than one ws-schemagen.jar depending on what features have been installed. We need to be more specific in our code to ensure we use the one in `/bin/tools` folder of the installed Liberty runtime.

https://github.com/OpenLiberty/liberty-language-server/blob/aa0ecd035927f4cad43d99f990402266a856cb40/lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java#L74

The schema generation fails with the following error if the `wrong` jar file is used.

```
Jul 27, 2022 3:08:12 PM com.sun.tools.jxc.SchemaGenerator$Runner compile
WARNING: Unrecognized options found: [/Users/cherylking/eclipse-workspace-testlibertytools/Sample/.libertyls/ol-22.0.0.6.xsd]
Jul 27, 2022 3:08:12 PM com.sun.tools.jxc.SchemaGenerator run
SEVERE: null
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

It would also be a good idea to update the cached schema file to the most recent LTS 22.0.0.6 for the early release.

### PR DESCRIPTION
**Pull request by mattbsox**
Fixes #80

### CODE REVIEW

File: lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java [hunk 1]
```diff
@@ -71,7 +72,7 @@ public String resolve(String baseLocation, String publicId, String systemId) {
           //Check workspace for Liberty installation and generate schema.xsd file
           //Return schema URI as String, otherwise use cached schema.xsd file
           if (libertyWorkspace.isLibertyInstalled()) {
-            Path schemaGenJarPath = LibertyUtils.findFileInWorkspace(libertyWorkspace, "ws-schemagen.jar");
+            Path schemaGenJarPath = LibertyUtils.findFileInWorkspace(libertyWorkspace, Paths.get("bin/tools/ws-schemagen.jar"));
```
Comment (cherylking) [inline lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java:75]: I thought it was supposed to be more something like `Paths.get("bin", "tools", "ws-schemagen.jar")`?

File: lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java [hunk 2]
```diff
@@ -204,7 +204,7 @@ private List<Feature> getInstalledFeaturesList(String documentURI) {
         return libertyWorkspace.getInstalledFeatureList();
       }
 
-      Path featureListJAR = LibertyUtils.findFileInWorkspace(documentURI, "ws-featurelist.jar");
+      Path featureListJAR = LibertyUtils.findFileInWorkspace(documentURI, "bin/tools/ws-featurelist.jar");
```
Comment (cherylking) [inline lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java:207]: Should change to use `Paths.get()` here also.

File: lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java [hunk 3]
```diff
@@ -88,17 +88,17 @@ public static URI getDocumentAsUri(DOMDocument document) {
      */
     public static Path findFileInWorkspace(String serverXmlURI, String filename) {
```
Comment (cherylking) [inline lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java:89]: I thought this one should be changed to take a `Path` as the second parameter also.

**Review by cherylking — CHANGES_REQUESTED** [review]

File: lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java (hunk 1 above)
Comment (mattbsox) [reply lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java]: Fixed.

File: lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java (hunk 2 above)
Comment (mattbsox) [reply lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java]: Fixed.

**Review by cherylking — APPROVED** [review]

**Pull request merged**